scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A script to check that travis.yml file & protractor.conf.js have the
+same test suites."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+import utils
+
+
+def extract_travis_jobs_section():
+    """Extracts the env/jobs section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of all jobs in travis.yml file.
+    """
+    travis_file = read_travis_yml_file()
+    jobs_raw = travis_file['env']['jobs']
+    suites_from_jobs = []
+    for job in jobs_raw:
+        # The jobs are in the form RUN_E2E_TESTS_ACCESSIBILITY=true
+        # so the following line removes the RUN_E2E_TESTS_(14 length)
+        # and =true(5 length) part.
```

Comment:
Use a regex and capture groups to capture this instead. That way, you don't need the 14 and -5 at all, and the intention is clear from the code. (I.e. the regex covers the "RUN_E2E_TESTS_...=true" part.)

Reply:
Done